tests/Equinox.CosmosStore.Integration/CosmosCoreIntegration.fs

```diff
@@ -294,11 +296,12 @@ type Tests(testOutputHelper) =
 
         let! expected = add6EventsIn2BatchesEx ctx streamName 4
 
+        let! res = Events.getAllBackwards ctx streamName 10L 1
         let! res =
-            Events.getAllBackwards ctx streamName 10L 1
-            |> AsyncSeq.concatSeq
-            |> AsyncSeq.takeWhileInclusive (fun x -> x.Index <> 4L)
-            |> AsyncSeq.toArrayAsync
+            res
+            |> TaskSeq.collectSeq id
+            |> TaskSeq.takeWhileInclusive (fun x -> x.Index <> 4L)
+            |> TaskSeq.toArrayAsync |> Async.AwaitTask
```

Comment:
```suggestion
            |> TaskSeq.toArrayAsync
            |> Async.AwaitTaskCorrect
```
?

Reply:
Didnt think that was available in the test context; now replaced in all prod and test code 👍